### ISSUE TITLE
Error: No module named 'sklearn.preprocessing.label'

### ISSUE THREAD

**Issue by hyrmet** [pplonski/my_ml_service#22]
Exception while loading the algorithms to the registry, No module named 'sklearn.preprocessing.label'

**Comment by tilias-um5s** [comment]
Dear hyrmet,

I think you are using a newer version of scikit learn which had several changes, try to use the same version as the author (scikit-learn==0.21.3) , and check if that helps. You can also read this post in stackoverflow: https://stackoverflow.com/questions/60305098/modulenotfounderror-no-module-named-sklearn-preprocessing-data

**Comment by pplonski** [comment]
@tilias-um5s you are right, that might be a solution.
